vlib/v/checker/tests/sum_type_infix_err.vv

```diff
@@ -0,0 +1,7 @@
+type Abc = int | string
+
+fn main() {
+	x := Abc(0)
+	_ := x + Abc(5)
+	_ := 'test' + x
```

Comment:
It looks like this pull may forbid infix operators on a pointer to a sum type, or an optional sum type.
Could you add a line e.g. `_ = unsafe{&x + 5}` to ensure no error message for that please?

Reply:
Ok, it fails. Do you have an idea how we could make it work?

Reply:
is_ptr check should help